pandas/tests/indexes/test_engine.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas._libs.index import (Int64Engine, UInt64Engine,
+                                Float64Engine, ObjectEngine)
+
+
+class TestNumericEngine(object):
+
+    @pytest.mark.parametrize('data', [[0, 1, 2]])
```

Comment:
why is this parameterized over data?

Reply:
Fixed. You start out with a certain structure and forget to check if is sensible for the end product...